e2e/etcd_release_upgrade_test.go

```diff
@@ -86,3 +87,58 @@ func TestReleaseUpgrade(t *testing.T) {
 		}
 	}
 }
+
+func TestReleaseUpgradeV2ToV3(t *testing.T) {
```

Comment:
i do not think we need the actual binary. this test is good in general for ensuring upgrade working.